modules/client-configuration/pages/contact-methods-saltssh-tunnel.adoc

```diff
@@ -0,0 +1,97 @@
+[[contact-methods-pushssh]]
+= SSH Push (with tunnel) contact method
+
+
+SSH Push (with tunnel) ([literal]``ssh-push-tunnel``) is used in environments where clients cannot reach the {productname} Server directly.
+In this environment, clients are located in a firewall-protected zone called a DMZ.
+No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
+
+This SSH method creates an encrypted tunnel from the {productname} Server on the internal network to the clients located on the DMZ.
+After all actions and events are executed, the tunnel is closed.
+
+The server uses SSH to contact the clients at regular intervals, checking in and performing scheduled actions and events.
+
+
+// FiXME: only here with tunnel, otherwise move to contact-methods-saltssh.adoc
+[IMPORTANT]
+====
+Re-installing systems using the provisioning model is not currently supported on clients managed with SSH Push.
+====
```

Comment:
I suppose this note makes sense for both ssh types with tunnel and without it. The problem here that the client itself can't initiate communication with the server, what is required for such operations.

Reply:
ok, I'll it to saltssh, too.